protostar/cheatable_starknet/controllers/contracts.py

```diff
@@ -209,7 +209,8 @@ async def execute_constructor_entry_point(
                 caller_address=0,
                 entry_point_type=EntryPointType.CONSTRUCTOR,
                 call_type=CallType.DELEGATE,
-                class_hash=class_hash_bytes,
+                class_hash=class_hash,
+                initial_gas=100000000000000000000000,
```

Comment:
[Use constant from cairo-lang instead of coming up with a number](https://github.com/software-mansion/protostar/pull/1526/commits/250c3bd0cd85b2ab0f41c7541aa46e7d91fe7bf4)